Libraries generated on windows fail to lint in *nix based systems

## Expected Behavior

Libraries generated should work on Windows and Unix based systems

## Current Behavior
Libraries generated on Windows machines fail to lint on Unix based systems

## Failure Information (for bugs)
error TS5058: The specified path does not exist: '/path/to\my-lib\tsconfig.lib.json'.
*note the backslashes in the path

Angular CLI version 8.0.3
@nrwl/workspace version 8.2.0

### Steps to Reproduce

From Windows:
Create new nx workspace, then
```
ng g @nrwl/workspace:library my-lib
```

From Unix system, checkout latest code containing lib and run:
```
yarn run lint
```

I fixed this by replacing the double "\\" characters with "/" in the angular.json file

--- a/packages/workspace/src/schematics/library/library.ts
+++ b/packages/workspace/src/schematics/library/library.ts
@@ -11,13 +11,13 @@ import {
   move,
   noop
 } from '@angular-devkit/schematics';
+import { join, normalize } from '@angular-devkit/core';
 import { Schema } from './schema';
 
 import { NxJson } from '@nrwl/workspace';
 import { updateJsonInTree, readJsonInTree } from '@nrwl/workspace';
 import { toFileName, names } from '@nrwl/workspace';
 import { formatFiles } from '@nrwl/workspace';
-import { join, normalize } from 'path';
 import { offsetFromRoot } from '@nrwl/workspace';
 
 export interface NormalizedSchema extends Schema {

